util/util.go

```diff
@@ -272,7 +272,7 @@ func LoadResource(location string, network bool, urls []string) ([]byte, error)
 		}
 		defer resp.Body.Close()
 		return ioutil.ReadAll(resp.Body)
-	} else if strings.HasPrefix(location, "/") {
+	} else if strings.HasPrefix(location, "/var/lib/rancher/conf") {
```

Comment:
Please don't change this method.  This is a generic method to load files.  I'd rather you check the condition up front in the enable code.